Makefile

```diff
@@ -79,12 +79,18 @@ fmt-py:
 	black .
 
 .PHONY: lint
-lint: lint-server lint-client
+lint: lint-local-server lint-hosted-server lint-client
 
 .PHONY: lint-server
-lint-server: fmt-py
-	flake8 server --per-file-ignores='server/test/fixtures/dataset_config_outline.py:F821 server/test/fixtures/server_config_outline.py:F821 server/test/performance/scale_test_annotations.py:E501'
+lint-server: lint-local-server lint-hosted-server
 
+.PHONY: lint-local-server
+lint-local-server: fmt-py
+	flake8 local_server --per-file-ignores='local_server/test/fixtures/dataset_config_outline.py:F821 local_server/test/fixtures/server_config_outline.py:F821 local_server/test/performance/scale_test_annotations.py:E501'
+
+.PHONY: lint-hosted-server
+lint-hosted-server: fmt-py
+	flake8 hosted_server --per-file-ignores='hosted_server/test/fixtures/dataset_config_outline.py:F821 hosted_server/test/fixtures/server_config_outline.py:F821 hosted_server/test/performance/scale_test_annotations.py:E501'
```

Comment:
Is `hosted_server` correct?  Or should it be `server`?  Ditto for other targets, eg, clean